### PR TITLE
Pin numpy to < 1.20

### PR DESCRIPTION
**Pull request by verveerpj**
**Issue**
Numpy >= 1.20 is not supported by python 3.6

### CODE REVIEW

**Review by jondequinor — DISMISSED** [review]

**Comment by verveerpj** [comment]
Tried pinning ert-storage to < 0.2, but that seemed to raise other issues. Maybe ert-storage has to pin numpy?

File: setup.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def package_files(directory):
         "graphlib_backport; python_version < '3.9'",
         "jinja2",
         "matplotlib",
-        "numpy",
+        "numpy < 1.20",
```
Comment (sregales-TNO) [inline setup.py:90]:
```
"numpy < 1.20; python_version < '3.7'"
"numpy; python_version >= '3.7'"
```
?

Comment (verveerpj) [reply setup.py:90]: No, as discussed on slack, pip does not anyway. And then we have the mypy failures again on > 3.6.

**Comment by sregales-TNO** [comment]
is ignoring the types a temporary solution?
otherwise what is the value of using a strict mypy configuration if we are going to ignore each new finding?

**Comment by verveerpj** [comment]
> is ignoring the types a temporary solution?
> otherwise what is the value of using a strict mypy configuration if we are going to ignore each new finding?

Yes, preferably we only have these in situations like these, that resolve themselves with the passage of time... (i.e. when we get rid of python 3.6)

**Review by sregales-TNO — APPROVED** [review]

**Pull request merged**